From 7768798964eb0e4f95eaecffb93b5d0ca28a38af Mon Sep 17 00:00:00 2001
From: Daniel Golle <daniel@makrotopia.org>
Date: Sat, 3 Jun 2017 20:00:03 +0200
Subject: [PATCH] MIPS: pci-mt7620: enabled PCIe on MT7688
To: linux-mips@linux-mips.org,
    John Crispin <john@phrozen.org>
Cc: Wei Yongjun <yongjun_wei@trendmicro.com.cn>,
    Ralf Baechle <ralf@linux-mips.org>,
    linux-mediatek@lists.infradead.org

Use PCIe support for MT7628AN also on MT7688.
Tested on WRTNODE2R.

Signed-off-by: Daniel Golle <daniel@makrotopia.org>
---
 arch/mips/pci/pci-mt7620.c | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/mips/pci/pci-mt7620.c
+++ b/arch/mips/pci/pci-mt7620.c
@@ -316,6 +316,7 @@ static int mt7620_pci_probe(struct platf
 		break;
 
 	case MT762X_SOC_MT7628AN:
+	case MT762X_SOC_MT7688:
 		if (mt7628_pci_hw_init(pdev))
 			return -1;
 		break;
